scripts/download_codeforces.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Script - download codeforces dataset from s3
+# options:
+# $1              specify a percentage of dataset used as train set
+# $2              specify a percentage of dataset used as test set
+# $3              specify a percentage of dataset used as validation set
+# $4              specify if developer mode is on, default: false
+# $5              specify if dataset needs to be shuffled, default: false
+# $6              specify a path to astminer .jar file
+# $7              specify a path to splitiing script
+
+TRAIN_SPLIT_PART=$1
+VAL_SPLIT_PART=$2
+TEST_SPLIT_PART=$3
+DEV=$4
+SHUFFLE=$5
+DATA_DIR=./data
+DATASET_NAME=cf
+ASTMINER_PATH=$6
+SPLIT_SCRIPT=$7
+
+DATA_PATH=${DATA_DIR}/${DATASET_NAME}
+
+if [ ! -d $DATA_DIR ]
+then
+  mkdir $DATA_DIR
+fi
+
+echo "Downloading dataset ${DATASET_NAME}"
+if [ -d "$DATA_PATH" ]
+then
+  echo "$DATA_PATH exists."
+else
+  if [ ! -f "$DATA_DIR/cf.zip" ]
+  then
+    aws s3 cp s3://datasets.ml.labs.aws.intellij.net/codeforces-code-clone/anti-plagiarism-datasets-master.zip "$DATA_DIR/cf.zip"
+  fi
+
+  echo "Unzip dataset"
+
+  if [ $DEV ]
+  then
+    echo "Dev mode"
+    unzip -qq "$DATA_DIR/cf.zip" "anti-plagiarism-datasets-master/rounds/1314,1315.zip" -d $DATA_DIR/
+  else
+    unzip "$DATA_DIR/cf.zip" -d $DATA_DIR
+  fi
+
+  mkdir $DATA_PATH
+
+  for round in $(find "$DATA_DIR/anti-plagiarism-datasets-master/rounds" -name "*.zip" -type f)
+  do
+    unzip "$round" -d "$DATA_DIR/anti-plagiarism-datasets-master/rounds"
+    round_dir="${round%.zip}"
+    find "$round_dir"/*  -type d -name "*[A-D]" -exec mv {} "$DATA_PATH" \;
+    rm -rf "$round_dir"
+    rm "$round"
+  done
+  rm -rf $DATA_DIR/anti-plagiarism-datasets-master
```

Comment:
Should we remove `.zip` too? Or I missed it?

Reply:
We removed it in line ` rm "$round"`. File `codeforces.zip` I suggest not to delete because it is quite a large file and downloading it can take quite a long time